client/state/selectors/get-site-computed-attributes.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Internal dependencies
+ */
+import {
+	getRawSite,
+	getSiteOption,
+	getSiteTitle,
+	isSitePreviewable,
+	getSiteDomain,
+	getSiteSlug,
+	isJetpackSite,
+	isSiteConflicting
+} from 'state/sites/selectors';
+import { getSiteOptions, canCurrentUser } from 'state/selectors';
+import { withoutHttp } from 'lib/url';
+
+/**
+ * Returns the site options
```

Comment:
This function doesn't return site options, it returns computed properties for the site object. Guessing copy/paste from `getSiteOptions`.